.travis.yml

```diff
@@ -17,22 +17,20 @@ cache:
 
 env:
   global:
-    - PATH="$HOME/.composer/vendor/bin:$PATH"
+    - PATH="$HOME/.local/bin:$HOME/.composer/vendor/bin:$PATH"
     - SYMFONY_DEPRECATIONS_HELPER=weak
 
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
-      env: CS_FIXER=run
     - php: 5.3
       env: COMPOSER_FLAGS="--prefer-lowest"
     - php: 5.6
-      env: SYMFONY_VERSION=2.3.*
+      env: SYMFONY_VERSION=2.3.* TEST_DOC=run
     - php: 5.6
       env: SYMFONY_VERSION=2.6.*
     - php: 5.6
-      env: SYMFONY_VERSION=2.7.*
+      env: SYMFONY_VERSION=2.7.* CS_FIXER=run
```

Comment:
I would prefer separate jobs for this. Clearer IMO.

BTW, fixer will be dropped when @rande will get some time to test #3273.

Same for `TEST_DOC` please.